core/src/main/java/me/aurium/beetle/branch/util/PreStoredHashSet.java

```diff
@@ -1,44 +1,53 @@
-package me.aurium.beetle.branch;
+package me.aurium.beetle.branch.util;
 
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Represents a set that already has an element in it
  */
-public class AlreadyStoredHashSet<T> {
+public class PreStoredHashSet<T> {
 
     private T alreadyStored;
     private final Set<T> otherThingsInTheSet;
+    private final boolean linked;
 
     //fuck code in the constructor rules
```

Comment:
There's no code in the constructor here?

Reply:
oh thats a remnant from a while ago, there used to be a bit of code there